### ISSUE TITLE
Add Fixtures /scores  for the 2018/2019 season

### ISSUE THREAD

**Issue by AchoArnold** [ManrajGrover/football-cli#63]
## Question  
I would love to use this application to see scores for the current 2018/2019 season. Is that possible?

**Comment by manrajgrover** [comment]
Yes, it is something that is possible. The issue is the base API this project uses has moved to `v2` recently and `v1` does not get those updated results. See #45 for more details. I had started migrating the project to `v2` but haven't completed it. #59

I would love to have your help in moving the project to `v2`.

**Comment by AchoArnold** [comment]
I would have really loved to help but unfortunately, javascript/node isn't my specialty. I'm more into PHP

**Comment by haroldduan** [comment]
Well done!🎉I am looking forward to your work!

**Comment by Freelancer-98** [comment]
I am looking into this. Let me know, if someone else is looking into it, as well.
I have modified the fixture command, working on other commands. Here's the result -
![football-standings](https://user-images.githubusercontent.com/22821480/50494703-e48fd580-0a4a-11e9-838a-ad45c437e32c.jpg)
